### ISSUE TITLE
client_config.ClientConfigBB::cfg.ClientConfig::flags’ may be used uninitialized

### ISSUE THREAD

**Issue by ZikZakFR** [fuzziqersoftware/newserv#131]
ProxyServer.cc reports an error:

```
newserv/src/ProxyServer.cc: In static member function ‘static void ProxyServer::UnlinkedSession::on_input(Channel&, uint16_t, uint32_t, std::string&)’:
newserv/src/ProxyServer.cc:281:27: error: ‘client_config.ClientConfigBB::cfg.ClientConfig::flags’ may be used uninitialized [-Werror=maybe-uninitialized]
  281 |         client_config.cfg.flags |= Client::Flag::IS_DC_V1;
      |         ~~~~~~~~~~~~~~~~~~^~~~~
newserv/src/ProxyServer.cc:265:18: note: ‘client_config’ declared here
  265 |   ClientConfigBB client_config;
      |                  ^~~~~~~~~~~~~
cc1plus: all warnings being treated as errors
make[2]: *** [CMakeFiles/newserv.dir/build.make:706: CMakeFiles/newserv.dir/src/ProxyServer.cc.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:83: CMakeFiles/newserv.dir/all] Error 2
make: *** [Makefile:146: all] Error 2

```

**Comment by fuzziqersoftware** [comment]
This should be fixed by 5fe21b8e.

**Comment by ZikZakFR** [comment]
> This should be fixed by [5fe21b8](https://github.com/fuzziqersoftware/newserv/commit/5fe21b8eecfee27b141d62d8517e5268e750794e).

Brilliant! No more error! Compilation succeeded.
